Sources/OpenCombine/Publishers/Publishers.Zip.swift

```diff
@@ -0,0 +1,756 @@
+//
+//  Publishers.Zip.swift
+//
+//  Created by Eric Patey on 29.08.2019.
+//
+
+#if canImport(COpenCombineHelpers)
+import COpenCombineHelpers
+#endif
+
+extension Publishers {
+
+    /// A publisher created by applying the zip function to two upstream publishers.
+    public struct Zip<UpstreamA, UpstreamB>: Publisher
+        where UpstreamA: Publisher,
+        UpstreamB: Publisher,
+        UpstreamA.Failure == UpstreamB.Failure
```

Comment:
```suggestion
    public struct Zip<UpstreamA: Publisher, UpstreamB: Publisher>: Publisher
        where UpstreamA.Failure == UpstreamB.Failure
```

The same goes for all other Zip publishers